newscoop/src/Newscoop/GimmeBundle/EventListener/PublicResourcesListener.php

```diff
@@ -15,15 +15,11 @@
 
 class PublicResourcesListener
 {
-    protected $em;
-    protected $serverService;
-    protected $security;
+    protected $container;
 
-    public function __construct($em, $serverService, $security)
+    public function __construct($container)
```

Comment:
i'v tested that - it takes little more memory, but in this case 3 ->get(*) calls on every request were more expensive.